logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnIncomingCall.kt

```diff
@@ -7,21 +7,20 @@ import com.wire.kalium.logic.callingLogger
 import com.wire.kalium.logic.data.call.CallMapper
 import com.wire.kalium.logic.data.call.CallRepository
 import com.wire.kalium.logic.data.call.ConversationType
-import com.wire.kalium.logic.data.id.FederatedIdMapper
-import com.wire.kalium.logic.data.id.toConversationId
+import com.wire.kalium.logic.data.id.QualifiedIdMapper
 import com.wire.kalium.logic.feature.call.CallStatus
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 
-//TODO(testing): create unit test
+// TODO(testing): create unit test
 class OnIncomingCall(
     private val callRepository: CallRepository,
     private val callMapper: CallMapper,
-    private val federatedIdMapper: FederatedIdMapper,
+    private val qualifiedIdMapper: QualifiedIdMapper,
     private val scope: CoroutineScope
 ) : IncomingCallHandler {
     override fun onIncomingCall(
-        conversationId: String,
+        conversationIdString: String,
```

Comment:
Could we rename this and other instances to `qualifiedConversationIdString` or `federatedConversationIdString` or `conversationIdWithDomainString` ? Otherwise it reads as if it was the legacy conversationId string w/o the domain and could generate confusion.

Reply:
It's not always a qualifiedConversationId because on non federated environment we will just receive conversation Id without domain